docs/design/bundle-sources.md

```diff
@@ -0,0 +1,273 @@
+
+<h3 id="proc-creating-a-fbc">
+Procedure
+</h3>
+
+1. Create a directory for the catalog by running the following command:
+
+    ```sh
+    mkdir <catalog_dir>
+    ```
+
+1. Create a Dockerfile that can build a catalog image by running the `opm generate dockerfile` command:
+
+    ```sh
+    opm generate dockerfile <catalog_dir>
+    ```
+
+    * The Dockerfile must be in the same parent directory as the catalog directory that you created in the previous step:
+        *Example directory structure*
+
+        ```sh
+        .
+        ├── <catalog_dir>
+        └── <catalog_dir>.Dockerfile
+        ```
+
+1. Populate the catalog with the package definition for your Operator by running the `opm init` command:
+
+    ```sh
+    opm init <operator_name> \
+    --output json \
+    > <catalog_dir>/index.json
+    ```
+
+    This command generates an `olm.package` declarative config blob in the specified catalog configuration file.
+
+## Adding a plain bundle to your file-based catalog
+
+Currently, the `opm render` command does not support adding plain bundles to catalogs. You must manually add plain bundles to your file-based catalog, as shown in the example below.
+
+<h3 id="prereqs-adding-a-plain-bundle-to-fbc">
+Prerequisites
+</h3>
+
+* `opm` CLI tool
+* A plain bundle image
+* A file-based catalog
+* Push access to a container registry, such as [Quay](https://quay.io)
+* Docker or Podman
+
+<h3 id="proc-adding-a-plain-bundle-to-fbc">
+Procedure
+</h3>
+
+1. Verify that your catalog's `index.json` or `index.yaml` file is similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {   
+        {
+         "schema": "olm.package",
+         "name": "<operator_name>",
+        }    
+    }
+    ```
+
+1. To create an `olm.bundle` blob, edit your `index.json` or `index.yaml` file, similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {
+        "schema": "olm.bundle",
+        "name": "<operator_name>.<version>",
+        "package": "<operator_name>",
+        "image": "quay.io/<organization_name>/<repository_name>:<image_tag>", 
+        "properties": [
+            {
+                "type": "olm.package",
+                "value": {
+                "packageName": "<operator_name>",
+                "version": "<bundle_version>"
+                }
+            },
+            {
+                "type": "olm.bundle.mediatype",
+                "value": "plain+v0"
+            }
+      ]
+    }
+    ```
+
+1. To create an `olm.channel` blob, edit your `index.json` or `index.yaml` file, similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {
+        "schema": "olm.channel",
+        "name": "<default_channel_name>",
+        "package": "<operator_name>",
+        "entries": [
+            {
+                "name": "<operator_name>/<bundle_version>"
+            }
+        ]
+    }
+    ```
```

Comment:
This is a step that I _really_ want newcomers to be able to completely skip, but I'm not sure we're there yet. In the meantime, I think I'd suggest changing `<default_channel_name>` to `<desired_channel_name>` (and we might need to provide an example of a reasonable channel name)

Reply:
@joelanford Sounds good. I changed it to `<desired_channel_name>` and mentioned about examples of the default channel names which could be `stable`, `fast` or `preview`. 
Let me know if that is what you were thinking of. Also, does the default channel name be followed by semver i.e. `stable:v0.0.1`?

Reply:
Perhaps just link to here: https://olm.operatorframework.io/docs/best-practices/channel-naming/

And then if you include an actual example, choose something like `candidate-v0`?

Reply:
That makes sense. I've linked to the channel naming conventions and also included this example in my [latest commit](https://github.com/operator-framework/operator-registry/pull/1127/commits/83cdaeab3ad29bdc44513d2816f890203f031c56)